drivers/flash/Kconfig.cadence_qspi_nor

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2022 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_CAD_QSPI_NOR := cdns,qspi-nor
+
+config FLASH_CAD_QSPI_NOR
+	bool "Cadence Quad SPI Flash driver"
+	depends on FLASH
+	default $(dt_compat_enabled,$(DT_COMPAT_CAD_QSPI_NOR))
+	select FLASH_HAS_PAGE_LAYOUT
+	select FLASH_HAS_DRIVER_ENABLED
+	help
+	  Enable Cadence QSPI-NOR support.
+
+config CAD_QSPI_MICRON_N25Q_SUPPORT
```

Comment:
wrap all of these with:

`if FLASH_CAD_QSPI_NOR`

Reply:
Resolved, Thanks.